middlewares/validators/user.js

```diff
@@ -213,11 +213,27 @@ const validateImageVerificationQuery = async (req, res, next) => {
   }
 };
 
+async function validateUpdateRoles(req, res, next) {
```

Comment:
min and max properties of the joi.object() schema are set to 1. This means that the schema only allows a single value for the member and archived properties. However, the function validateUpdateRoles expects a user to specify either the member or archived property, but not both

Reply:
yes